packages/core/notarization/commitments.go

```diff
@@ -100,10 +103,14 @@ func (f *EpochCommitmentFactory) ECR(ei epoch.Index) (ecr epoch.ECR, err error)
 
 	root := make([]byte, 0)
 	conflict1 := make([]byte, 0)
+	conflict1a := make([]byte, 0)
+	conflict1b := make([]byte, 0)
 	conflict2 := make([]byte, 0)
 
-	conflict1Hashed := blake2b.Sum256(append(append(conflict1, epochRoots.tangleRoot[:]...), epochRoots.stateMutationRoot[:]...))
-	conflict2Hashed := blake2b.Sum256(append(append(conflict2, epochRoots.stateRoot[:]...), epochRoots.manaRoot[:]...))
+	conflict1aHashed := blake2b.Sum256(append(append(conflict1a, epochRoots.tangleRoot[:]...), epochRoots.stateMutationRoot[:]...))
+	conflict1bHashed := blake2b.Sum256(append(append(conflict1b, epochRoots.stateRoot[:]...), epochRoots.manaRoot[:]...))
+	conflict1Hashed := blake2b.Sum256(append(append(conflict1, conflict1aHashed[:]...), conflict1bHashed[:]...))
+	conflict2Hashed := blake2b.Sum256(append(conflict2, epochRoots.activityRoot[:]...))
```

Comment:
Rename all these variables to "branch*", it was a false positive of renaming all branches -> conflicts =P